examples/platforms/efr32/uart.c

```diff
@@ -39,155 +39,145 @@
 
 #include "utils/code_utils.h"
 
-#include "bspconfig.h"
-#include "em_cmu.h"
-#include "em_gpio.h"
-#include "em_usart.h"
+#include "em_core.h"
+#include "uartdrv.h"
+
+#include "retargetserialhalconfig.h"
 
 enum
 {
-    kPlatformClock     = 32000000,
-    kBaudRate          = 115200,
-    kReceiveBufferSize = 128,
+    kReceiveFifoSize = 128,
 };
 
-static void processReceive(void);
-static void processTransmit(void);
+#define USART_INIT                                                                           \
+    {                                                                                        \
+        RETARGET_UART,                                    /* USART port */                   \
+        115200,                                           /* Baud rate */                    \
+        RETARGET_TX_LOCATION,                             /* USART Tx pin location number */ \
+        RETARGET_RX_LOCATION,                             /* USART Rx pin location number */ \
+        (USART_Stopbits_TypeDef)USART_FRAME_STOPBITS_ONE, /* Stop bits */                    \
+        (USART_Parity_TypeDef)USART_FRAME_PARITY_NONE,    /* Parity */                       \
+        (USART_OVS_TypeDef)USART_CTRL_OVS_X16,            /* Oversampling mode*/             \
+        false,                                            /* Majority vote disable */        \
```

Comment:
Spinel docs say flow control must be used. I had framing errors at higher baud rates if it's not enabled.

Tested and works if:

- Change `uartdrvFlowControlNone` to `uartdrvFlowControlHwUart`
- Change `0,0` at the end of this struct to `RETARGET_CTS_LOCATION, RETARGET_RTS_LOCATION`

Reply:
@tomas-c, thanks!  I updated this PR with your proposed changes.